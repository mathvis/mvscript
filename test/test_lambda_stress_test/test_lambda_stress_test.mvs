let a: lambda[string, int, float, bool, vector, point, matrix, [lambda[]]]lambda[]int;
