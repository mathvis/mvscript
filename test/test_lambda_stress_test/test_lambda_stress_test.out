Decl (Variable (VarIdentifier "a") (Just (LambdaT [StringT,IntT,FloatT,BoolT,VectorT,PointT,MatrixT,ArrayT (LambdaT [] VoidT)] (LambdaT [] IntT))) Nothing)
